lib/index.js

```diff
@@ -391,17 +391,13 @@ Nightwatch.prototype.runProtocolAction = function(requestOptions, callback) {
 Nightwatch.prototype.addError = function(message, logMessage) {
   var currentTest;
   if (this.api.currentTest) {
-    currentTest = '[' + Utils.getTestSuiteName(this.api.currentTest.module) + ' / ' + this.api.currentTest.name + ']';
+    currentTest = '[' + Utils.getTestSuiteName(this.api.currentTest.module) + (this.api.currentTest.name ? (' / ' + this.api.currentTest.name) : '') + ']';
```

Comment:
Why would the currentTest.name be null or defined ? If i am not mistaken this is the test suite or test file being executed.